src/utils/route_tracker.py

```
@@ -0,0 +1,79 @@
+import json
+import folium
```

Ruff (0.8.2)

`folium` imported but unused

Remove unused import: `folium`

(F401)